gym/lib/gym/generators/build_command_generator.rb

```diff
@@ -40,6 +40,7 @@ def options
         options << "-archivePath #{archive_path.shellescape}"
         options << "-derivedDataPath '#{config[:derived_data_path]}'" if config[:derived_data_path]
         options << "-resultBundlePath '#{result_bundle_path}'" if config[:result_bundle]
+        options << "OTHER_SWIFT_FLAGS='-Xfrontend -debug-time-function-bodies' | grep .[0-9]ms | grep -v ^0.[0-9]ms | sort -nr > culprits.txt '#{config[:analyze_build_time]}'" if config[:analyze_build_time]
```

Comment:
This will overwrite existing value of `OTHER_SWIFT_FLAGS` set in Xcode project.
To preserve existing build settings, this approach is preferred:

```
OTHER_SWIFT_FLAGS="\$(inherited) -Xfrontend -debug-time-function-bodies"
```

Also, should `grep`s and output redirect be inserted here?
Maybe it would be more clear/safer to let gym do the build, then post-process raw build log to collect all function build times?